docs/lib/geo/fragments/js/search.md

```diff
@@ -26,23 +16,17 @@ Install the necessary dependencies with the following command:
 npm install -S @maplibre/maplibre-gl-geocoder maplibre-gl maplibre-gl-js-amplify
```

Comment:
Can we remove installation of `maplibre-gl-js` here since we removed it from the imports?

Reply:
You mean `maplibre-gl`? We cannot remove that because it is listed as a `peerDependency` of `maplibre-gl-js-amplify` and not a hard dependency, thus we want to have developers manually install it.